test/parallel/test-http-set-trailers.js

```diff
@@ -103,7 +103,7 @@ server.on('listening', function() {
     headers: {}
   }, function(res) {
     res.on('end', function() {
-      //console.log(res.trailers);
+      // console.log(res.trailers);
       assert.ok('x-foo' in res.trailers, 'Client doesn\'t see trailers.');
```

Comment:
Suggestion: remove the comment and change the error message to include `res.trailers`.